doc/api/http.md

```diff
@@ -747,6 +747,10 @@ added: v0.1.90
 Begin accepting connections on the specified `port` and `hostname`. If the
 `hostname` is omitted, the server will accept connections on any IPv6 address
 (`::`) when IPv6 is available, or any IPv4 address (`0.0.0.0`) otherwise.
+
+*Note*: in most operating systems, listening to any IPv6 address (`'::'`) may
```

Comment:
Maybe say `to the unspecified IPv6 address` instead of `to any IPv6 address`. I know it results in listening on all addresses, but saying `any addresses` suggests that it's true for other addresses like `::1`.